### PR TITLE
KAFKA-3397: use -1(latest) as time default value for tools.GetOffsetShell

### PR DESCRIPTION
**Pull request by vesense**
minor fix for prompt the user, or we will get a error message:

> Missing required argument "[time]"

### CODE REVIEW

**Comment by vesense** [comment]
cc @guozhangwang

**Comment by SinghAsDev** [comment]
Either we can provide a default value of time, say always latest, or if we want to mandate users to specify it, we should put a check to make sure user provides time, if not print usage and die.

**Comment by guozhangwang** [comment]
I agree with @SinghAsDev that we can provide a default value, for example latest; this would make more sense after KIP-33 is in so that the searching can be more accurate.

**Comment by vesense** [comment]
Thanks @SinghAsDev @guozhangwang for your reply. I agree with you that we can use -1(latest) as the default value. i.e. add line:
`.defaultsTo(-1)`

File: core/src/main/scala/kafka/tools/GetOffsetShell.scala [hunk 1]
```diff
@@ -43,10 +43,11 @@ object GetOffsetShell {
                            .describedAs("partition ids")
                            .ofType(classOf[String])
                            .defaultsTo("")
-    val timeOpt = parser.accepts("time", "timestamp of the offsets before that")
+    val timeOpt = parser.accepts("time", "Timestamp of the offsets before that")
```
Comment (SinghAsDev) [inline core/src/main/scala/kafka/tools/GetOffsetShell.scala]: NIT: may be it is better to be inline with other param descriptions, which does not start with caps.

Comment (vesense) [inline core/src/main/scala/kafka/tools/GetOffsetShell.scala]: ok

**Comment by vesense** [comment]
cc @SinghAsDev

**Comment by guozhangwang** [comment]
Unit test passed locally. LGTM and merged to trunk.

**Pull request closed without merging**